aspnetcore/migration/70-80.md

```diff
@@ -73,64 +73,541 @@ In the project file, update each [Microsoft.AspNetCore.*](https://www.nuget.org/
 
 ## Blazor
 
-We hope to have the Blazor section of this article updated soon.<!--In the meantime, see the following resources: -->
+<!-- UPDATE 8.0 Add API cross-links after RTM -->
 
-### Adopt .NET 8 features
+The following migration scenarios are covered:
 
-After following the guidance earlier in this article to update an app to 8.0, adopt specific features by following the links in <xref:aspnetcore-8#blazor>.
+* [Update a Blazor Server app](#update-a-blazor-server-app)
```

Comment:
The minimum you need to do to update a Blazor Server app is just update the TFM and the packages, which is covered by the general ASP.NET Core migration guidance above. This section is really about taking a Blazor Server app and turning it into a Blazor Web App, so I think it should have the corresponding heading, although I prefer "update" to "convert".

```suggestion
* [Convert a Blazor Server app into a Blazor Web App](#convert-a-blazor-server-app-into-a-blazor-web-app)
```